static/templates/actors/partials/item-line.html

```diff
@@ -1,5 +1,5 @@
 <li class="list-row">
-    <div class="item {{#if item.isTemporary}}temporary-item{{/if}}" data-item-id="{{item.id}}" data-item-is-container="{{isContainer}}" data-item-rarity="{{item.data.traits.rarity}}">
+    <div class="item {{#if item.isTemporary}}temporary-item{{/if}}" data-item-id="{{item.id}}" data-item-is-container="{{isContainer}}" data-item-rarity="{{item.data.data.traits.rarity}}">
```

Comment:
```suggestion
    <div class="item {{#if item.isTemporary}}temporary-item{{/if}}" data-item-id="{{item.id}}" data-item-is-container="{{isContainer}}" data-item-rarity="{{item.system.traits.rarity}}">
```
:o)

Reply:
I forgot that because its a document, we can do this without repercussions pre v10.